api/envoy/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto

```diff
@@ -229,25 +229,25 @@ message TcpProxy {
       [(validate.rules).duration = {gte {nanos: 1000000}}];
 
   // .. attention::
```

Comment:
this still doesnt work - i think it needs an extra/empty comment line above

this is how it renders to rst:

```rst

flush_access_log_on_connected
  (`bool <https://developers.google.com/protocol-buffers/docs/proto#scalar>`_) .. attention::
    This field is deprecated in favor of
    :ref:`flush_access_log_on_connected
    <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.TcpAccessLogOptions.flush_access_log_on_connected>`.
    Note that if both this field and :ref:`flush_access_log_on_connected
    <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.TcpAccessLogOptions.flush_access_log_on_connected>`
    are specified, the former (deprecated field) is ignored.

```

Reply:
thanks for rendering! I have now also locally parsed the protodoc to rst, and the block is indeed inlined after the type. I'll add an empty comment above the admonition!

Reply:
On Message it does work, but apparently not on members: 
a top-level .. attention:: https://github.com/envoyproxy/envoy/blob/020871ba10a301557191f5c4b1faa6671e1726ec/api/envoy/config/route/v3/route_components.proto#L2322 renders as https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto.html#config-route-v3-headermatcher

Reply:
its possible our templates are chomping ws differently

Reply:
Ok, the order has to change, first a description of the field and then the admonition. I see in other examples that the body of the comment is taken and inlined before the admonition block. I'll see if that is feasible in this case.